packages/cli/src/commands/init/init.js

```diff
@@ -90,6 +93,90 @@ async function createFromExternalTemplate(
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, Do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        await execa('sudo', ['gem', 'install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
+      } catch (err) {
+        throw new CLIError(
+          'Error occurred while trying to install CocoaPods, please install it manually.',
+          err,
+        );
+      }
+
+      try {
+        loader.start('Installing pods');
+      } catch (err) {
+        throw new CLIError(
+          'Failed to run "pod install", please try to run it manually.',
+          err,
+        );
+      }
+    }
+  }
+
+  try {
```

Comment:
This shouldn't be here, right? It's outside of the if statement.

Reply:
It should. The first `try...catch` is for handling if `cocoapods` is installed, if it is it goes straight to this one that install the pods, if not it goes to the installation of `cocoapods` that then triggers the loader with `Installing pods` and the actual command on line 141.

Reply:
Ah, right! Then... shouldn't the "loader.start('Installing pods');" be on line 141, between both try/catch'es?

Reply:
Can be, I didn't put it there because I didn't want to bug the user with another loading if they are both part of the dependencies, it only shows if the user has been prompted to install cocoapods.

Do you want to show it every time it runs `pod install`?

Reply:
Ah, right! Makes sense. When `pod` is already there, we will transparently install it w/o showing the loader.

I like that

Reply:
Great!